extensions/adobe/experience/adcloud/searchadvertisingaccount.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertisingaccount",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Search Advertising Account defines Customer Configuration on Search Advertising Platform.",
+  "definitions": {
+    "searchadvertisingaccount": {
+      "properties": {
+        "xdm:accountId": {
```

Comment:
Why cant we use id that comes out of common record class here ?

Reply:
Most of the entities (search) have two identifiers - one is ad-cloud identifier which uniquely identifies the record across search engines and other is external identifier (unique per search engine). _id that comes out of record is used as an internal identifier.